opentelemetry-sdk/tests/metrics/export/test_export.py

```diff
@@ -518,3 +519,15 @@ def test_push_controller(self):
         controller.shutdown()
         self.assertTrue(controller.finished.isSet())
         exporter.shutdown.assert_any_call()
+
+    def test_push_controller_suppress_instrumentation(self):
```

Comment:
It would be nice to have a test to verify that the token exists during the export. I think you can do this by asserting the order of the calls.